code/array/array-new-vs-fixnum-times-map.rb

```diff
@@ -0,0 +1,17 @@
+require "benchmark/ips"
+
+ELEMENTS = 9
```

Comment:
No problems @nateberkopec. :-) Is a comment in the PR sufficient, or do you want me to add it to the `README`?